### PR TITLE
Feature slam

### PR DESCRIPTION
**Pull request by TBragg**
Let's do this! I don't think I have permissions to publish. Bumped the version number though...

### CODE REVIEW

File: src/scripts/slam.coffee [hunk 1]
```diff
@@ -26,7 +25,151 @@ images = [
...
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
+  "http://media1.giphy.com/media/IfaFEvfGz8CXK/giphy.gif",
```
Comment (melieru) [inline src/scripts/slam.coffee:57]: I think this fails the golden DRY rule

Comment (cmckendry) [inline src/scripts/slam.coffee:57]: I'll allow it

Comment (prestifidibs) [inline src/scripts/slam.coffee:57]: image

**Comment by TBragg** [comment]
Let's build this shiiiit

**Pull request merged**